arm/Microsoft.Automation/automationAccounts/.parameters/parameters.json

```diff
@@ -30,7 +30,7 @@
         "runbooks": {
             "value": [
                 {
-                    "name": "TestRunbook",
+                    "name": "TestRunbook01",
```

Comment:
why the renaming?

Reply:
Already changed bacl to its original value